### PR TITLE
Translated readme.

### PR DESCRIPTION
**Pull request by davidhsrtoyota**
Translation complete.
Please verify.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 tmc_gazebo_plugins
 ===========================
 
-概要
+Overview
 ===========================
-[Gazebo](http://gazebosim.org/)用のロボットに関わらず使えるプラグインを管理する。
+Manages the plugin that can be used regardless of the [Gazebo](http://gazebosim.org/) robot.
```
Comment (yuka-hashiguchi) [inline README.md]: The original sentence means
 * This repository manages the plugin for Gazebo
 * The plugin is not depend on robots (I mean, it's "general" plugin. Not only for some specific robots.)
Could you describe these info in here?

**Review by yuka-hashiguchi — COMMENTED** [review]
please let me know if my comment is unclear. :bow:

File: README.md (hunk 1 above)
Comment (davidhsrtoyota) [reply README.md]: How about this:

- This repository manages the plugin that can be used with any [Gazebo](http://gazebosim.org/) robot.

Comment (yuka-hashiguchi) [reply README.md]: perfect!

Comment (davidhsrtoyota) [reply README.md]: OK. :)
I will make the edit and repush.

**Review by yuka-hashiguchi — APPROVED** [review]
Thank you for your quick response!

**Pull request merged**